### ISSUE TITLE
Enter and Exit Custom Message

### ISSUE THREAD

**Issue by PinoyLifeCraft** [rlf/uSkyBlock#667]
This is what i Translate that message!
![tl](https://cloud.githubusercontent.com/assets/13610530/13062295/06510930-d49a-11e5-996e-ff3b87367706.png)
I get the Successful Translation **Only on that player**
![2016-02-16_06 45 48](https://cloud.githubusercontent.com/assets/13610530/13062331/566dfa0e-d49a-11e5-878e-765291521854.png)
If i go to another player's island **I receive the default Message**
![2016-02-16_06 47 18](https://cloud.githubusercontent.com/assets/13610530/13062353/97de6708-d49a-11e5-9a9a-11a4b52f1174.png)

**Comment by dutchy1001** [comment]
This happens as the WorldGuard region is not updated yet till that player comes online.

**Comment by PinoyLifeCraft** [comment]
So when he will online it will works?
[SkyBlock] -> - Dutchy1001's SkyBlock

**Comment by dutchy1001** [comment]
yes when the player gets online it will change after a few seconds

**Comment by dutchy1001** [comment]
did "john2004" log on to the server??

**Comment by PinoyLifeCraft** [comment]
john2004 is me and the picture U saw is mine then HANCY in that time is not [+] Online
